test/mitmproxy/proxy/test_mode_specs.py

```diff
@@ -54,6 +54,7 @@ def test_parse_specific_modes():
     assert ProxyMode.parse("socks5")
     assert ProxyMode.parse("dns").resolve_local
     assert ProxyMode.parse("dns:reverse:8.8.8.8")
+    assert ProxyMode.parse("dtls:reverse:127.0.0.1:0")
```

Comment:
Let's use a valid destination port here in case we ever get to tightening the check.